e2e/scripts/actions/env_path.test

```diff
@@ -0,0 +1,30 @@
+#!/singularity/test
+# env path tests
+
+source common.func
+
+default_image="docker://alpine:3.8"
+default_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+
+custom_image="docker://godlovedc/lolcow"
+custom_path="/usr/games:${default_path}"
+partial_path="/foo"
+overwritten_path="/usr/bin:/bin"
+
+stdout-contains "DefaultPath"           "${default_path}"                   singularity exec ${default_image} printenv PATH
```

Comment:
what does a failing test look like? do these `testing` functions have verbose output?

Reply:
When it fails : https://github.com/sylabs/singularity/pull/3486/files/e714d276a7150107570c967490ca95b1e8bca17c#diff-8281e19ba9c3129e27bece1c9222ad0bR84